core/src/components/checkbox/test/basic/checkbox.e2e.ts

```diff
@@ -141,4 +141,21 @@ configs({ modes: ['ionic-md'], directions: ['ltr'] }).forEach(({ title, screensh
       await expect(checkboxes).toHaveScreenshot(screenshot(`checkbox-small`));
     });
   });
+
+  test.describe(title('checkbox: basic visual tests'), () => {
+    test('should have a Invalid visual applied correctly', async ({ page }) => {
+      await page.setContent(
+        `
+        <div id="checkboxes">
```

Comment:
We should add some padding to this container so the captured screenshot for the focus state includes the entire outline. 

For example:
```html
<div id="checkboxes" style="padding: 8px">
```

Would be good to double check the padding value to make sure it is large enough.

Reply:
I did test this, and it should be enough, thanks for the headsup